1.9/administration/monitoring/metrics/reference.md

```diff
@@ -0,0 +1,108 @@
+---
+post_title: Metrics Statistics Reference
```

Comment:
is this really the right title? aren't they just metrics, not metrics statistics... metrics attributes, maybe?